src/MCMCChains.jl

```diff
@@ -19,7 +20,7 @@ import Tables
 import TableTraits
 import IteratorInterfaceExtensions
 
-using LinearAlgebra: diag, dot, BlasReal
+using LinearAlgebra: diag, dot, BlasReal,eigmax
```

Comment:
It seems this change is not needed. I'd prefer the existing version that adds fewer things to the namespace:
```suggestion
using LinearAlgebra: diag, dot, BlasReal
```